sidebars.js

```diff
@@ -16,6 +16,47 @@ const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
   mainSidebar: [{type: 'autogenerated', dirName: '.'}],
 
+  termtableSidebar: [
+    'index',
+    {
+      type: 'category',
+      label: 'osu! 中英翻译术语表',
+      collapsed: false,
+      items: [
+        'termtable/general',
+        'termtable/beatmap',
+        'termtable/gameplay',
+        'termtable/client',
+        'termtable/people',
+        'termtable/tech',
+        'termtable/rules',
+      ]
+    }
+  ],
+
+  lazerSidebar: [
+    'index',
+    {
+      type: 'category',
+      label: '从零开始的 osu!lazer 入门指南',
```

Comment:
We already have an index page for `osu!lazer 指南`, try to clarify a clearer order.

Reply:
I placed 'index', at the top of the sidebar because adding the 文档主页 page to both the 术语表 and osu!lazer指南 categories makes it more intuitive for users to navigate through our documentation. This way, users can easily return to the main page containing all the docs by clicking 文档主页 on the sidebar.

I have used the same writeup in `termtableSidebar` as well.

---

As for the index page for both `术语表` and `osu!lazer指南`，I'm still looking for a way which we can link these pages onto the collapsable tabs. Which is: by clicking `osu! 中英翻译术语表` tab, the sub-categories will appear and the current page will be switched into those index pages.

Those index pages can only be accessed via navbar for now.

Reply:
I remembered an available way for this, on official document:

```js
              link: {
                type: 'doc',
                id: 'guides/docs/sidebar/index',
              },
```

Reply:
Fixed in [this commit](https://github.com/osu-atri/osu-dictionary/pull/25/commits/01bd0c37da2a98546f1541afa281999a68f825f2). Marked as resolved now.